runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -231,10 +231,13 @@ else if (id == ItemID.PLATINUM_TOKEN)
 
 	private String stackValueText(int qty, int gePrice, int haValue, int haProfit)
```

Comment:
You could just change geprice/havalue/haprofit to `long` and that would promote the multiplications later

Reply:
I thought of that as well but didn't want to change the function header if I didn't have to for some reason. I'll make this change there isn't really any reason not to change it.